zenml/cli/pipeline.py

```diff
@@ -31,3 +37,106 @@ def set_metadata_store():
     click.echo('Comparing pipelines in repo: Starting app..')
```

Comment:
The  name of the function should not be `set_metadata_store`?

Reply:
Changed to `compare_pipelines` in commit f1bc3bb.